app/main.py

```diff
@@ -2,0 +1,16 @@
+from app.errors import VaccineError, NotWearingMaskError
+from app.cafe import Cafe
```

Comment:
According to the code style in the checklist, it's better to use parentheses when importing multiple classes. Consider changing these lines to follow this style.